### DISCUSSION TITLE
Helper: "Helper" is lost in the configuration menu when the "Helper" is disabled.

### DISCUSSION THREAD

**Discussion by vtb77** [home-assistant/frontend#11066] [Other feature requests]
When, from the configuration menu, I disable a "Helper" of the type "input_select", the "Helper" disappears from the "Helper" configuration window. I cannot use it again. In the "Helper" configuration window I can't find any option to see the disabled "Helpers".
So to remember the "Helper" I have to do it manually.

To do it manually, I edit the file "`\config\.storage\core.entity_registry`" and change the field "`"disabled_by":"user",`" change "`"user"`" to "`null`". I restart Home Assistant and it reappears.

I understand that it is a bug.

Thank you, good job and happy new year 2022.

**Comment by karwosts** [comment]
While adding support for this on the helpers menu would be nice, you can find your disabled helpers under the "entities" tab, and re-enable them that way.
